config.json

```diff
@@ -11,7 +11,7 @@
     "bob",
     "sum-of-multiples",
     "strain",
-    "point-mutations",
+    "hamming",
```

Comment:
Please move the old `"point-mutations"` to the `"deprecated"` section at the bottom of the file `config.json` as well.